src/__tests__/components/auth/Register.test.js

```diff
@@ -1,30 +1,42 @@
 import Enzyme from 'enzyme';
 import EnzymeAdapter from 'enzyme-adapter-react-16';
-import React from "react";
-import { shallow, mount } from "enzyme";
-import configureStore from "redux-mock-store";
-import thunk from "redux-thunk";
-import Register from '../../../components/auth/Register';
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { Register } from '../../../components/auth/Register';
+
 Enzyme.configure({ adapter: new EnzymeAdapter() });
 
 describe('<Register/>', () => {
-  const middlewares = [thunk];
-  const mockStore = configureStore(middlewares);
-  const initialUserState ={
-    user: {}
-  };
+  const editor = shallow(<Register registerUser={jest.fn()} />);
 
-  const store = mockStore({ ...initialUserState });
-  const initialState = {
-    name: '',
-    email: '',
-    password: '',
+  it('should render without crashing', () => {
+    expect(editor).toMatchSnapshot();
+  });
+  const props = {
+    registerUser: (newUser, history) => jest.fn(),
+    auth: { isAuthenticated: false, user: {} },
     errors: {}
   };
-   const editor = shallow(<Register registeruser={jest.fn()} />);
-  
-  it("should render without crashing", () => {
-    expect(editor).toMatchSnapshot();
+
+  it('test Register component onchange', () => {
+    const wrapper = shallow(<Register {...props} />);
+    const instance = wrapper.instance();
+    console.log('instance ++++++++++++++++', instance);
```

Comment:
REmove this `console.log`

Reply:
I have removed console.log('instance ++++++++++++++++', instance);